site/public/mjs/sql-toolbox.js

```diff
@@ -25,6 +25,7 @@ export async function runSqlQuery() {
         if (json.status !== 'success') {
             error_mesage.textContent = json.message;
             error.style.display = 'block';
+            document.getElementById('download-sql-btn').disabled = true;
```

Comment:
Let's move this line to the top of the function. We should disable this button for so long as it takes us to get and generate the results to avoid any possible run conditions with downloading (i.e. if I hit "run query" and while the network request is in progress, I hit "download csv", what is going to get downloaded? The original dataset or possibly the new one as it loads in?).

Moving this to the top would also allow getting rid of the duplicate of this line in the exception catch as we know it's `true` except for when we get to the very bottom of this try.